puppet/modules/secretsgit/manifests/init.pp

```diff
@@ -0,0 +1,22 @@
+class secretsgit(
+  String $group = 'secretsgit',
+  Stdlib::Absolutepath $path = '/srv/secretsgit',
+  Array[String] $users = []
+) {
+
+  group { $group:
+    ensure => present
+  }
+
+  file { $path:
+    ensure => directory,
+    owner => 'root',
+    group => $group,
+    mode => '2770',
```

Comment:
Can you align the parameters?

Reply:
I thought that's not a thing anymore in Puppet land? But I sure can.